apps/mull-api/src/app/user/user.service.ts

```diff
@@ -73,4 +75,87 @@ export class UserService {
     await this.userRepository.save(currentUser);
     return true;
   }
+
+  async removeFriend(currentUserId: number, userIdToRemove: number): Promise<boolean> {
+    if (currentUserId === userIdToRemove) {
+      throw new UnprocessableEntityException('Cannot remove oneself as friend.');
+    }
```

Comment:
Should we also check that `userIdToRemove` is an actual user. Noticed we're not checking in the `addFriend` function as well which is a more serious issue as I believe it could break the UI.

Reply:
I think its not necessary, it would add an extra query (I think), and I don't see why we should send back an error if we want to remove someone from your friends who never was your friend to begin with

Reply:
Yes, I think it's overkill as well. Lets keep it as is.